e2e/__tests__/circusDeclarationErrors.test.ts

```diff
@@ -12,8 +12,8 @@ import runJest from '../runJest';
 
 skipSuiteOnJasmine();
 
-it('defining tests and hooks asynchronously throws', () => {
-  const result = runJest('circus-declaration-errors', [
+it('defining tests and hooks asynchronously throws', async () => {
+  const result = await runJest('circus-declaration-errors', [
```

Comment:
This `runJest` function actually doesn't return a promise, so we should find some way to test this kind of stuff.
This test got failed after I put `runtime.requireModule` into a async function.

From:

```js
const esm = runtime.unstable_shouldLoadAsEsm(testPath);

if (esm) {
  await runtime.unstable_importModule(testPath);
} else {
    runtime.requireModule(testPath);
}
```

To:

```js
await localRequire(testPath);
```

Reply:
`runJest` spawns `jest` in a shell, so it should not be affected by any internal changes in Jest